src/renderer/vt/vtrenderer.hpp

```diff
@@ -150,6 +150,8 @@ namespace Microsoft::Console::Render
         bool _delayedEolWrap{ false };
 
         [[nodiscard]] HRESULT _Write(std::string_view const str) noexcept;
+        [[nodiscard]] HRESULT _WriteFormattedString(const size_t estimatedSize, const std::string* const pFormat, ...) noexcept;
+        [[nodiscard]] HRESULT _WriteFormattedString(const size_t estimatedSize, const std::string* const pFormat, va_list args) noexcept;

```

Comment:
I'm surprised this compiles; I would expect an overload with a `...` to conflict with an overload with an argument!

Reply:
Yeah it seemed vaguely weird to me but it was working. However, if I take the guessing out, then it'll collapse back to one and there's no concern here.